### ISSUE TITLE
0006972:
update to extjs 6

### ISSUE THREAD

**Issue by Gloirin** [tine20/tine20#3533]
**Reported by pschuele on 21 Aug 2012 13:33**

update to extjs 6

**Additional information:** ext4+ stuff not yet resolved

- extFixes: MultiSelectionModel gibt&#039;s nicht mehr
- Ext.ux.tree.CheckboxSelectionModel
- DisplayLayout -&gt; no more form layout
- boot splash element
- legacy app on
- state provider
- Ext.ux.printer getXtype overrride =&gt; xtype / ref etc &lt;=
- progressbar fix
- css login dialog
- state weenier state!
- recheck eventproxy in Ext.ux.WindowFactory::getCenterPanel
- westpanel state
- geoExt
        {
          &quot;text&quot;: &quot;GeoExt.js&quot;,
          &quot;path&quot;: &quot;../library/GeoExt/script/&quot;
        },
    {
          &quot;text&quot;: &quot;MapPanel.js&quot;,
          &quot;path&quot;: &quot;js/widgets/&quot;
        },

Ext.ux.PercentCombo

    {
          &quot;text&quot;: &quot;HorizontalFitLayout.js&quot;,
          &quot;path&quot;: &quot;js/ux/layout/&quot;
        },
        {
          &quot;text&quot;: &quot;CenterLayout.js&quot;,
          &quot;path&quot;: &quot;js/ux/layout/&quot;
        },
        {
          &quot;text&quot;: &quot;RowLayout.js&quot;,
          &quot;path&quot;: &quot;js/ux/layout/&quot;
        },

**Comment by Gloirin** [comment]
Depends on #2283

**Comment by Gloirin** [comment]
Depends on #2266

**Comment by Gloirin** [comment]
Depends on #2228

**Comment by Gloirin** [comment]
Depends on #2186

**Comment by Gloirin** [comment]
Depends on #1168

**Comment by Gloirin** [comment]
Depends on #2957

**Comment by Gloirin** [comment]
Depends on #2863

**Comment by Gloirin** [comment]
Depends on #2758

**Comment by Gloirin** [comment]
Depends on #2753

**Comment by Gloirin** [comment]
Depends on #2072

**Comment by Gloirin** [comment]
Depends on #3434

**Comment by Gloirin** [comment]
Depends on #3173

**Comment by Gloirin** [comment]
Depends on #2721

**Comment by Gloirin** [comment]
Depends on #2605

**Comment by Gloirin** [comment]
Depends on #265

**Comment by Gloirin** [comment]
Depends on #1302

**Comment by Gloirin** [comment]
Depends on #2251

**Comment by Gloirin** [comment]
Depends on #2250

**Comment by Gloirin** [comment]
Depends on #3338

**Comment by Gloirin** [comment]
Depends on #3463

**Comment by Gloirin** [comment]
Depends on #3337

**Comment by Gloirin** [comment]
Depends on #3196

**Comment by Gloirin** [comment]
Depends on #2723

**Comment by Gloirin** [comment]
Depends on #3527

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 23 Aug 2012 07:51**

Just as information - perhaps you can go on with the migration if the compatibility layer is complete ;)

http://www.sencha.com/blog/ext-js-3-to-4-migration/

http://stackoverflow.com/questions/6909513/switching-from-extjs3-to-extjs4-worth-the-hassle

**Comment by Gloirin** [comment]
**Comment posted by shlberlin on 30 Jun 2014 06:44**

+1

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 2 Jul 2014 15:24**

http://docs.sencha.com/ext/5.0.0/whats_new/5.0/whats_new.html

**Comment by Gloirin** [comment]
**Comment posted by mspahn on 18 Jul 2014 13:32**

I checked this out and searched a bit.
There is no know way to &quot;update&quot; or easy migrate to ExtJS 5.

The most devs are rewriting their application for ExtJS 5 using the best practise.

The API change is just a way too big.

It could be a start to create a new ExtJS 5 base for Tine 2.0 and create a skeleton app for each tine 2.0 module.

Then migrate the &quot;logic&quot; to the new skeleton, this might work, but it will be a lot of work.

**Comment by Gloirin** [comment]
**Comment posted by ravermeister on 22 Feb 2017 21:15**

...meanwhile
http://docs.sencha.com/extjs/6.2.1/guides/quick_start/introduction.html

:)

**Comment by Gloirin** [comment]
**Comment posted by travm1 on 7 Apr 2017 23:40**

Here is the wiki for ExtJS: https://www.everipedia.com/Ext_JS/

**Comment by Gloirin** [comment]
**Comment posted by shochdoerfer on 14 May 2017 19:32**

Not sure if this makes sense or not but we have but a kind of bridge to combine ExtJS6 with ExtJS3, ExtJS4 or ExtJS5 in one application allowing a migration of individual components over time. Ping me for a demo if that makes sense for you.

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 15 May 2017 06:38**

hi Stephan, we&#039;ll talk about this in our next meeting. thanks for your offer :)

**Comment by pschuele** [comment]
no longer relevant, I suppose.
